allennlp/common/testing/checklist_test.py

```diff
@@ -0,0 +1,34 @@
+from typing import Optional
+from checklist.test_suite import TestSuite
+from checklist.test_types import MFT
+from allennlp.sanity_checks.task_checklists.task_suite import TaskSuite
+
+
+@TaskSuite.register("fake-task-suite")
+class FakeTaskSuite(TaskSuite):
```

Comment:
Is this for our internal testing, or would users also use this?

Reply:
It's for our testing.

Reply:
Hmm I'm a little conflicted about having this here. I view the `common.testing` module as a place for general testing utilities that other libraries might want to use for their tests as well. So if this is potentially useful for other libraries, then let's definitely keep it here, otherwise I suggest we move this into the `tests/sanity_checks/task_checklists/` directory somewhere.